Source/Plugin.BLE/Shared/KnownServices.cs

```diff
@@ -63,6 +63,7 @@ public static KnownService Lookup(Guid id)
             new KnownService("TI SensorTag Connection Control", Guid.ParseExact("f000ccc0-0451-4000-b000-000000000000", "d")),
             new KnownService("TI SensorTag OvertheAir Download", Guid.ParseExact("f000ffc0-0451-4000-b000-000000000000", "d")),
             new KnownService("TXRX_SERV_UUID RedBearLabs Biscuit Service", Guid.ParseExact("713d0000-503e-4c75-ba94-3148f18d941e", "d")),
+            new KnownService("CITYSPORTS Treadmill", Guid.ParseExact("00001826-0000-1000-8000-00805f9b34fb", "d")),
```

Comment:
The BLE [Assigned Numbers](https://www.bluetooth.com/wp-content/uploads/Files/Specification/HTML/Assigned_Numbers/out/en/Assigned_Numbers.pdf?v=1706903097464) document lists the ID 0x1826 as "Fitness Machine service" (which is also how nRF connect shows it). I think it would be better to use this official and generic name, instead of using a specific vendor name (which is only one of many who may implement it).

Reply:
Honestly, because I have no idea where these GUIDs come from and the fact that you had a hardcoded list made me think they were just manufacturer generated rather than assigned... I'll tweak it. 

As for the rest, I'm not adding them because I've not found any use for them or been able to work out what they actually do, so I don't want to put my name against something I can't attest to

Reply:
> Honestly, because I have no idea where these GUIDs come from and the fact that you had a hardcoded list made me think they were just manufacturer generated rather than assigned... I'll tweak it.

thx

> As for the rest, I'm not adding them because I've not found any use for them or been able to work out what they actually do, so I don't want to put my name against something I can't attest to

Very reasonable attitude 😃, but at least for those that are included in the BLE Assigned Numbers document (and listed as "known" characteristics by nRF connect) there is no need to worry: They are standard characteristics with a well-defined name. If you are unsure what they do or how they are supposed to be used, maybe the Fitness Machine service specs will help to clarify that: https://www.bluetooth.com/specifications/specs/fitness-machine-service-1-0/

For those listed as "Unknown Characteristic" by nRF Connect, it's certainly a good idea to be a bit more careful: They are probably non-standard / vendor-specific characteristics, so maybe let's rather ignore them for now (unless you find some documentation that clearly defines their function).

Reply:
Well, that's a handy document I wish I'd known about last week before I reversed the entire protocol with pcaps... I'll add the rest if I get a chance, not sure when that'll be, maybe merge what we have here for now?

Reply:
Nevermind, I'll just finish just up myself. No big deal ...

Reply:
I'll simply merge this now. Let me know if you find any problem in hindsight.